src/ui/router/Browse.js

```diff
@@ -68,6 +68,7 @@ define([
                 objects = objects.reverse();
 
                 openmct.router.path = objects;
```

Comment:
I think we should move this to `src/ui/router/ApplicationRouter.js` there is existing `setPath` may be doing something different

Reply:
This code has been here for a long time. We can move it, but probably have a separate PR to do it.